docs/configure/admission/configuration.md

```diff
@@ -18,56 +21,20 @@ By default, the Admission Controller will monitor the following resources:
   * ReplicationControllers
 
 If you'd like to add additional resources, you can use the `rules`
-setting on the Helm chart:
+setting on the Helm chart. Adding this to the `values.yaml` when installing the Insights Agent:
 ```yaml
-rules:
-- apiGroups:
-  - custom
-  apiVersions:
-  - v1
-  operations:
-  - CREATE
-  - UPDATE
-  resources:
-  - customResource
-  scope: Namespaced
-```
-
-## Policies
-You can fine-tune which policies are applied by the admission controller. Any policy with a `high` or `critical` severity will instruct the Admission Controller to deny the deployment. 
-
-Specifically, the following auditing tools
-can be enabled or disabled as part of admission control:
-* Polaris - checks for security and best practices
-* OPA - apply custom policies to resources ([see docs](/configure/policy/policy))
-* Pluto - disallow resources that have been deprecated
-
-Enforce policies from the above auditing tools ([see docs](/configure/admission/enforce))
-
-To enable or disable a particular report, run:
-```bash
-curl -X POST https://insights.fairwinds.com/v0/organizations/$org/admission/reports/$report \
-  -H "Authorization: Bearer $token" \
-  -d '{"enabled": false}'
-```
-
-where:
-* `$report` is one of `polaris`, `opa`, or `pluto`
-* `$org` is your organization's name in Insights
-* `$token` is the admin token found on your organization settings page
-
-### Polaris
-You can also upload a custom
-[Polaris configuration](https://polaris.docs.fairwinds.com/customization/checks/)
-to set which checks are marked as `danger`, and will therefore cause a workload to be rejected.
-
-
-You can also use the Polaris configuration to write
-[custom checks using JSON Schema](https://polaris.docs.fairwinds.com/customization/custom-checks/)
-
+admission:
+  insights-admission:
```

Comment:
I believe `insights-admission` goes at the top level, not inside `admission`

Reply:
Yep! Thank you. Fixing